src/controller/review.py

```diff
@@ -13,7 +13,7 @@
 login_manager = Shared.login_manager
 
 
-@review.route('/review', methods=['GET'])
+@review.route('/api/review', methods=['GET'])
```

Comment:
Backend is still expecting calls at `/review`. Why the change? Same for all below

Reply:
routing /review to the frontend and the backend simultaneously is impossible

Reply:
Wait my bad, this is the backend.